bin/testcase.py

```diff
@@ -231,17 +238,15 @@ def validate_format(
             flags = args if flags is None else flags + args
 
             ret = validator.run(self, mode=mode, constraints=constraints, args=flags)
-            if ret.ok is not True and ret.ok != config.RTV_WA:
-                bar.log(f"Expected exit code {config.RTV_AC} or {config.RTV_WA}, got {ret.ok}")
-                ret.ok = config.RTV_WA
-            ok = ret.ok == True
+            if ret.status == ExecStatus.ERROR:
+                bar.log(f"Unxpected exit code!")
 
-            validator_accepted.append(ok)
-            message = validator.name + (' accepted' if ok else ' rejected')
+            validator_accepted.append(True if ret.status else False)
```

Comment:
Tiny optimization using the boolean coercion :smile:
```suggestion
            validator_accepted.append(bool(ret.status))
```